### PR TITLE
chore(client): Update readme and rust example

### PR DESCRIPTION
**Pull request by vovkman**
In relation to #252 load balancers sometimes require that client sends a ping. Updating the readme to call this out and including an example in the rust examples

### CODE REVIEW

File: examples/rust/src/bin/client.rs [hunk 1]
```diff
@@ -549,6 +550,23 @@ async fn geyser_subscribe(
...
+                            .send(SubscribeRequest {
+                                ping: Some(SubscribeRequestPing { id: 1 }),
+                                slots: request.slots.clone(),
+                                accounts: request.accounts.clone(),
+                                transactions: request.transactions.clone(),
+                                entry: request.entry.clone(),
+                                blocks: request.blocks.clone(),
+                                blocks_meta: request.blocks_meta.clone(),
+                                commitment: request.commitment.clone(),
+                                accounts_data_slice: request.accounts_data_slice.clone(),
```
Comment (fanatid) [inline examples/rust/src/bin/client.rs]:
```suggestion
                                ping: Some(SubscribeRequestPing { id: 1 }),
                                ..Default::default()
```
if `ping` field is defined all other fields are ignored, so we can set them to default value

Comment (vovkman) [reply examples/rust/src/bin/client.rs]: Oh right I was testing on a branch that didn't have the latest ping changes

Comment (vovkman) [reply examples/rust/src/bin/client.rs]: Updated, sorry for delay

Comment (fanatid) [reply examples/rust/src/bin/client.rs]: np, thanks for the update!

File: examples/rust/src/bin/client.rs [hunk 2]
```diff
@@ -524,7 +524,8 @@ async fn geyser_subscribe(
     request: SubscribeRequest,
     resub: usize,
 ) -> anyhow::Result<()> {
-    let (mut subscribe_tx, mut stream) = client.subscribe_with_request(Some(request)).await?;
+    let (mut subscribe_tx, mut stream) =
+        client.subscribe_with_request(Some(request.clone())).await?;
```
Comment (fanatid) [inline examples/rust/src/bin/client.rs]:
```suggestion
        client.subscribe_with_request(Some(request)).await?;
```

`.clone()` is not required now

Comment (vovkman) [reply examples/rust/src/bin/client.rs]: updated

**Comment by fanatid** [comment]
Thank you!

**Pull request merged**